components/checkbox/Checkbox.tsx

```diff
@@ -145,7 +145,7 @@ class Checkbox extends React.PureComponent<CheckboxProps, {}> {
           className={checkboxClass}
           ref={this.saveCheckbox}
         />
-        {children !== undefined && <span>{children}</span>}
+        {!!children && <span>{children}</span>}
```

Comment:
`0` will not be displayed.

Reply:
> `0` will not be displayed.

Yes it won't, but my understanding is it shouldn't either, if 0 is passed to a label, shouldn't it go in as a string? I do understand that this might break a lot of existing code, but it should be considered for type safety.

Reply:
If you have any other new ideas, you can make new PR to us.